src/Jackett.Common/Definitions/cartoonchaos.yml

```diff
@@ -126,6 +122,18 @@ search:
           args: category
     title:
       text: "{{ if .Result.title_torrent }}{{ .Result.title_torrent }}{{ else }}{{ .Result.title_text }}{{ end }}"
+      filters:
+        #  #11775
+        - name: re_replace # Seasons X-Y --> S0X-S0Y
+          args: ["(?i)\\bSeasons?[-.\\s]*(\\d)[-.\\s]*(?:-|to)[-.\\s]*(\\d)\\b", "S0$1-S0$2"]
+        - name: re_replace # Seasons X-YY --> S0X-SYY
+          args: ["(?i)\\bSeasons?[-.\\s]*(\\d)[-.\\s]*(?:-|to)[-.\\s]*(\\d+)\\b", "S0$1-S$2"]
+        - name: re_replace # Seasons XX-YY --> SXX-SYY
+          args: ["(?i)\\bSeasons?[-.\\s]*(\\d+)[-.\\s]*(?:-|to)[-.\\s]*(\\d+)\\b", "S$1-S$2"]
+        - name: re_replace # Season X --> S0X
+          args: ["(?i)\\bSeasons?[-.\\s]*(\\d)\\b", "S0$1"]
```

Comment:
the comment above the regex suggests that this should be a single season, but the regex is using seasons
typo in either the comment or the regex ?
same for the next one.

Reply:
oh, I'm half asleep . the ? in front of the s makes it optional.
still the comment is misleading. perhaps # Season(s) X --> S0X as the comment?

Reply:
Found the same `Seasons 1` on CRT for a single result. Worth adding there?